contributions/project-tenets.md

```diff
@@ -0,0 +1,56 @@
+# MRTK3 Tenets
+
+The following are the guiding principles by which the MRTK3 project abides. This is a living document and the list may be modified over time.
+
+1. [Break large changes into manageable chunks](#break-large-changes-into-manageable-chunks)
+1. [Semantic versioning](#semantic-versioning)
+1. [Support all application stages](#support-all-application-stages)
+1. [Centralized project authority](#centralized-project-authority)
+1. [Get community feedback on large decisions](#get-community-feedback-on-large-decisions)
+1. [Avoid backchannel code reviews](#avoid-backchannel-code-reviews)
+1. [Community contributions are the responsibility of the project](#community-contributions-are-the-responsibility-of-the-project)
+1. [Set clear project guidelines](#set-clear-project-guidelines)
+1. [Be prompt with feedback and anticipated shipping updates](#be-prompt-with-feedback-and-anticipated-shipping-updates)
+1. [Be kind and professional](#be-kind-and-professional)
+
+## Break large changes into manageable chunks
```

Comment:
formatting nit: Most markdown style guides recommend spaces between headers and their content (for example, https://google.github.io/styleguide/docguide/style.html#add-spacing-to-headings)

Reply:
added